kahu/deployment_guide.md

```diff
@@ -10,6 +10,8 @@ tags: ["deployment guide", "kahu", "soda-cdm"]
 Kahu features can be realised through deployment on kubernetes(k8s) cluster. 
 If you do not have a k8s cluster, please refer [kubernetes setup documentation](https://kubernetes.io/docs/setup) to setup one.
 
+Note: Need to install the required things for nfs server on the k8s nodes (Example: sudo apt install nfs-kernel-server on ubuntu)
```

Comment:
Is this necessary as we use NFS Server from docker image with all dependencies installed?